notebook/app.py

```diff
@@ -234,7 +234,7 @@ class JupyterNotebookApp(NotebookConfigShimMixin, LabServerApp):
     app_version = version
     extension_url = "/"
     default_url = Unicode("/tree", config=True, help="The default URL to redirect to from `/`")
-    file_url_prefix = "/notebooks"
+    file_url_prefix = "/tree"
```

Comment:
So the idea of using `/tree` is that there is already a redirect handler to open either with `/edit` or `/notebooks`:

https://github.com/jupyter/notebook/blob/7f1a2f49ef77abdca2dd22c42a4fcfb8e6890a11/notebook/app.py#L127-L160